public-ui/src/app/views/ehr-view/ehr-view.component.html

```diff
@@ -265,7 +265,7 @@ <h6 *ngIf="searchText.value.length > 0" class="primary-display medline-link">Int
                     </div>
                   </ng-container>
                   <ng-template #NonRolledUp>
-                    <div class="row" *ngIf="r.vocabularyId === 'SNOMED'">
+                    <div class="row">
```

Comment:
Sources graph must be shown irrespective of the vocabulary id (for non-rolled up concepts).